assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+{
+    "radio": {
+        "twr": "Kansai Tower",
+        "app": "Kansai Approach",
+        "dep": "Kansai Radar"
+    },
+    "icao": "RJBB",
+    "iata": "KIX",
+    "magnetic_north": -7,
+    "ctr_radius": 80,
+    "ctr_ceiling": 7000,
+    "initial_alt": 3000,
+    "position": ["N34d26m03.00", "N135d13m58.00"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N34d26m03.00", "N135d13m58.00"],
+    "has_terrain": false,
+    "wind": {
+		"angle": 204,
+        "speed": 9
+    },
```

Comment:
Be careful to not use "hard tabs" (the `\t` character). As shown in the github code preview thing here, `angle` and `speed are misaligned, as caused by there being two "hard tabs" before `angle`, while `speed` has eight spaces. The latter is preferred.

Please review the file and remove any tabs and replace with four spaces. I'm not sure how to do so in vscode, but sublime and atom both can search by regex, so if that's possible just find all `\t` and replace with four spaces.